en/sql/tuning.rst

```diff
@@ -2393,6 +2395,279 @@ The above **SELECT** query's plan is printed out as below; we can see "(sort lim
 
 .. _query-cache:
 
+Join Elimination Optimization
+-----------------------------
+
+The join elimination optimization is an optimization that removes the join for the table with 1 if there is no reference to a column of the table with 1 in the join in a 1:N relationship.
```

Comment:
Instead of using the 1:N terminology, we can explain a database relationship using the parent-child relationship terminology, where the parent entity is on the "one" (/1) side of a relationship with another table, and the child entity is on the "many" (/N/*) side of the relationship with the parent entity.

Using the parent-child relationship terminology, we can easily understand that the parent entity is the one with the primary key, while the child entity references the primary key in its own foreign key column. Therefore the terminology can help avoid confusion instead of "1" and "N".